hack/dev/kind/install.sh

```diff
@@ -202,7 +202,7 @@ main() {
     echo "And we are done :): "
 }
 
-while getopts "Ggpcrb" o; do
+while getopts "RGgpcrb" o; do
```

Comment:
RGgpcrb what this means 😄

Reply:
it's the flags, it's how getopt libc call works and the bash version is a shell wrapper around it https://www.gnu.org/software/libc/manual/html_node/Example-of-Getopt.html